CISS/transforms/utils.py

```diff
@@ -0,0 +1,9 @@
+def get_file_path(name, filenames):
+    for filename in filenames:
+        if name.upper() == filename[:-4].upper():
```

Comment:
Claudia and I caught up about this but in case anyone else see's this, [:-4] removes the file extension (i.e. '.csv') from the filename. In my next commit, I will make this more clear